modules/admission-webhook-deploy.adoc

```diff
@@ -0,0 +1,373 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-plug-ins.adoc
+
+[id="admission-webhook-deploy_{context}"]
+= Deploying a webhook server
+
+This procedure outlines high-level steps to deploy a webhook server as a DaemonSet and to deploy the webhook server as an aggregated API server. Configuring the webhook server as an aggregated API server allows other {product-title} components to communicate with the webhook using internal credentials and facilitates testing using the `oc` command. This also enables role based access control (RBAC) into the webhook and prevents token information from other API servers from being disclosed to the webhook. The API server calls the Kubernetes API server with its fully authenticated service account. The aggregated API server then calls to the webhook using the request header certificate. The webhook verifies incoming client certificates through the published request header aggregation CA.
+
+.Prerequisites
+
+* An {product-title} account with cluster administrator access.
+* The {product-title} CLI (`oc`) installed.
+* An existing, published webhook server container image.
```

Comment:
"existing" seems repetitive. "A published webhook server container image." should be sufficient.

Reply:
Resolved.